roles/kubernetes-apps/ansible/templates/nodelocaldns-second-daemonset.yml.j2

```diff
@@ -39,6 +39,10 @@ spec:
             cpu: {{ nodelocaldns_cpu_requests }}
             memory: {{ nodelocaldns_memory_requests }}
         args: [ "-localip", "{{ nodelocaldns_ip }}", "-conf", "/etc/coredns/Corefile", "-upstreamsvc", "coredns", "-skipteardown" ]
+        ports:
+        - containerPort: 9255
+          name: metrics
+          protocol: TCP
```

Comment:
It's OK  :-)
Referencing https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml#L158.